### ISSUE TITLE
Deploy Maven Build

### ISSUE THREAD

**Issue by NishkarshRaj** [NishkarshRaj/Maven-Using-CMD#20]
Use GitHub Actions to deploy the Maven build to remote repository (preferably Nexus) if static code analysis using SonarQube passes which is referenced in #17

**Comment by acodebreaker** [comment]
if analysis has some issues but the sonar scan is successful , we can still deploy , right ?
correct me if i am wrong

**Comment by NishkarshRaj** [comment]
@acodebreaker Exactly! You are right!

Quality Gates in SonarQube are set as parameters.
Suppose, we create a quality gate that allows a maximum of 10 bugs.

So, if there are 5 bugs, even though our build is not perfect, Quality Gate is passed, and we deploy it to the production environment 😄

**Comment by Avijit-07** [comment]
Hi @NishkarshRaj Can you kindly check your Personal access token has read & write access on the repo? 
I'm getting below 400 error while doing a mvn deploy.
Failed to transfer file https://maven.pkg.github.com/NishkarshRaj/Maven-Using-CMD/pkg1/mvn-cmd/1.0-SNAPSHOT/maven-metadata.xml with status code 400

**Comment by NishkarshRaj** [comment]
@Avijit-07 I have not given my personal access token yet.
I will give it later by today after my examination concludes 😞

**Comment by Avijit-07** [comment]
Sure man. No worries. In the meantime I'll check if I can deploy it with my own personal token or not. Thanks for the quick response.

**Comment by NishkarshRaj** [comment]
@Avijit-07 Yes, I have verified my access token is having the said permissions!
I have sent you a collaborator request for this repository which will provide you with the permissions to use your token as well.

**Comment by Avijit-07** [comment]
Thanks for that. Couldn't check yesterday due to some other work. I'll give it try today. Cheers

**Comment by NishkarshRaj** [comment]
@Avijit-07 do try and let me know the progress 😄
